code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -71,8 +71,8 @@
 
 	for (var/type as anything in childrentypesof(/datum/contextAction/flockdrone))
 		src.contexts += new type
-
-	src.AddComponent(/datum/component/flock_protection, FALSE, FALSE, FALSE)
+	APPLY_ATOM_PROPERTY(src, PROP_ATOM_FLOCK_THING, src)
+	src.AddComponent(/datum/component/flock_protection)
```

Comment:
```suggestion
	src.AddComponent(/datum/component/flock_protection, FALSE, FALSE, FALSE, FALSE)
```
Since Flockdrones actually have their own way of reporting all four types.